config/crd/bases/iam.services.k8s.aws_roles.yaml

```diff
@@ -56,6 +56,10 @@ spec:
               description:
                 description: A description of the role.
                 type: string
+              inlinePolicies:
+                additionalProperties:
+                  type: string
+                type: object
```

Comment:
Here is where it would be nice to have custom documentation, so that the CRD (and therefore the API reference) has information about expectations for the name of the key

Reply:
yuuuup. :)